create-service-and-ensure-existence.sh

```diff
@@ -0,0 +1,18 @@
+organization=$1
```

Comment:
THis should start with `#! /bin/bash -e`

`-e` is important so that it will fail on error.